### PR TITLE
Update parameter names for several functions

### PR DESCRIPTION
**Pull request by mkindahl**
This updates the parameter names for functions changed in
timescale/timescaledb#2472.

### CODE REVIEW

**Review by svenklemm — APPROVED** [review]

File: api.md [hunk 1]
```diff
@@ -403,7 +403,7 @@ still work on the resulting hypertable.
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of table to convert to hypertable |
+| `relation` | Identifier of table to convert to hypertable |
```
Comment (erimatnor) [inline api.md:406]: Assuming that punctuation is our standard:

```suggestion
| `relation` | Identifier of table to convert to hypertable. |
```

File: api.md [hunk 2]
```diff
@@ -614,7 +614,7 @@ when creating distributed hypertables.
...
```
Comment (erimatnor) [inline api.md:617]:
```suggestion
| `relation` | Identifier of table to convert to hypertable. |
```

File: api.md [hunk 3]
```diff
@@ -958,7 +958,7 @@ specified one.
 
 |Name|Description|
 |---|---|
-| `hypertable_or_cagg` | Hypertable or continuous aggregate from which to drop chunks.
+| `relation` | Hypertable or continuous aggregate from which to drop chunks.
```
Comment (erimatnor) [inline api.md:961]:
```suggestion
| `relation` | Hypertable or continuous aggregate from which to drop chunks. |
```

File: api.md [hunk 4]
```diff
@@ -3265,7 +3265,7 @@ All sizes are in bytes.
 
 |Name|Description|
 |---|---|
-| `main_table` | (REGCLASS) Name of the hypertable |
+| `hypertable` | (REGCLASS) Hypertable to show stats for |
```
Comment (erimatnor) [inline api.md:3268]:
```suggestion
| `hypertable` | Hypertable to show stats for. |
```

File: api.md [hunk 5]
```diff
@@ -3394,7 +3394,7 @@ information as a separate row per node.
 
 |Name|Description|
 |---|---|
-| `main_table` | (REGCLASS) Name of the hypertable |
+| `hypertable` | (REGCLASS) Hypertable to show detailed size of |
```
Comment (erimatnor) [inline api.md:3397]:
```suggestion
| `hypertable` | Hypertable to show detailed size of. |
```

File: api.md [hunk 6]
```diff
@@ -3473,7 +3473,7 @@ output of `hypertable_detailed_size` function.
 
 |Name|Description|
 |---|---|
-| `main_table` | (REGCLASS) Name of the hypertable |
+| `hypertable` | (REGCLASS) Hypertable to show size of |
```
Comment (erimatnor) [inline api.md:3476]:
```suggestion
| `hypertable` | Hypertable to show size of. |
```

**Review by erimatnor — APPROVED** [review]
Added some nits for consistency. I looks like some API function parameters document the type while many don't. Do we want to make this consistent too?

File: api.md (hunk 4 above)
Comment (mfreed) [reply api.md:3268]: I believe the "preferred" format is to include type argument's TYPE (we aren't great here, but I guess would like to trend towards better).

File: api.md (hunk 1 above)
Comment (mfreed) [reply api.md:406]: Is there a reason you don't just use "table" here?  I mean, I guess `relation` is correct from the format database semantics perspective, but I think will be confusing for the average user....

Comment (mkindahl) [reply api.md]: > Is there a reason you don't just use "table" here? I mean, I guess `relation` is correct from the format database semantics perspective, but I think will be confusing for the average user....

We had a discussion in the team and decided to use `hypertable` when it was a hypertable, and `relation` for other cases (e.g., a Postgres table, hypertable or cagg, etc.)

**Pull request merged**